go.mod

```diff
@@ -132,4 +132,7 @@ require (
 	k8s.io/kube-openapi v0.0.0-20211115234752-e816edb12b65 // indirect
 	sigs.k8s.io/json v0.0.0-20211020170558-c049b76a60c6 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.0 // indirect
+	github.com/blang/semver/v4 v4.0.0 // indirect
+	sigs.k8s.io/kubebuilder-release-tools/notes v0.0.0-20220428224951-d8a44c7aef35 // indirect
```

Comment:
These dependencies must go in the `hack/tools` module, then there you can perhaps add a Makefile target to pull down the release note binary and a target in the root makefile to generate notes.

Reply:
changes to this file need to be reverted

Reply:
done